client/pages/Dashboard/index.jsx

```diff
@@ -156,13 +175,18 @@ Dashboard.propTypes = {
   location: PropTypes.shape({ pathname: PropTypes.string.isRequired }).isRequired,
   loadActiveUser: PropTypes.func.isRequired,
   createEvent: PropTypes.func.isRequired,
+  getCategoryList: PropTypes.func.isRequired,
 };
 
-const mapStateToProps = state => ({ activeUser: state.activeUser });
+const mapStateToProps = state => ({
+  activeUser: state.activeUser,
+  socialClubs: state.socialClubs,
+});
 
 export default connect(mapStateToProps,
   {
```

Comment:
This `dispatch` object literal is getting long already, let's abstract to a method like `mapDispatchToProps`

Reply:
Gotcha!

Reply:
Can you give more context

Reply:
https://github.com/AndelaOSP/Andela-Socials/wiki/REACT-Best-Practices---Actions. Check this out. It might help.

Reply:
@kenneth-kip from the link you shared which is the guide, I have not implemented any of the " `Avoid this` ". Going by Bolaji's comment, its a matter of style and space, because `bindActionCreators` do the same work, you might want to check https://stackoverflow.com/questions/41342540/what-is-difference-between-dispatch-and-bindActionCreators

Going by your previous comment on another PR about consistency, I feel my implementation corresponds with what is already on the codebase, maintaining consistency. However, I will change to `bindActionCreators` using `mapDispatchToProps`